src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue8845.cs

```diff
@@ -0,0 +1,27 @@
+﻿#if WINDOWS
```

Comment:
Does it make sense to run this test on Android and iOS as well?

We typically let these tests run on all platforms even if the original issue is only happening on iOS. This also lets us make sure that we have consistent behavior between all platforms.

If it's broken on iOS and/or Android then log a bug and make a note of the bug inside this test.

Also does it make sense to address @jfversluis 's comment while we're at it?

https://github.com/dotnet/maui/pull/24276/files#r1731389699

Reply:
I have modified the visual test to run on all platforms. Could you please confirm once, anything else needs to be updated from our end.